beast/plotting/plot_toothpick_details.py

```diff
@@ -123,3 +124,60 @@ def plot_toothpick_details(asts_filename, seds_filename, savefig=False):
         fig.savefig("{}.{}".format(basename, savefig))
     else:
         plt.show()
+
+
+def fetch_filters(filename):
+    """
+    For any FITS table file (gst, ast, etc.), collect the filters based on the
+    columns containing "VEGA". Returns a sorted list of the full BEAST filter
+    names, assuming the camera is HST_WFC3.
+
+    Parameters
+    ----------
+    filename : str
+        filename containing VEGA measurements
+
+    Returns
+    -------
+    filters : list
+        list of full BEAST filter names
+
+    """
+
+    data = Table.read(filename)
+
+    # extract every filter mentioned in the table
+    # find all columns mentioning VEGA
+    filters = [f.split("_")[0] for f in data.columns if "VEGA" in f]
+
+    # sort the list of filters based on wavelength
+    # i.e. anything < 200nm is in the IR
+    rep = 0
+    for n in range(len(filters)):
+        if "L" in filters[n - rep]:
+            num = (int(filters[n - rep][-5:-2]))
+
+        else:
+            num = (int(filters[n - rep][-4:-1]))
+
+        if num < 200:
+            filters.append(filters.pop(filters.index('{0}'.format(filters[n - rep]))))
+            rep += 1
+
+    # add camera prefix
+    filters = ["HST_WFC3_" + f for f in filters]
+
+    # check to see that the filters actually exist
+    source = "{0}/vega.hd5".format(__ROOT__)
```

Comment:
Why not directly use the filters file?

__default__ = __ROOT__ + "/filters.hd5"

Reply:
See https://beast.readthedocs.io/en/latest/beast_setup.html#beast-filters